manufacturing-ai-suite/industrial-edge-insights-time-series/docs/user-guide/get-started.md

```diff
@@ -77,9 +77,11 @@ cd edge-ai-suites/manufacturing-ai-suite/industrial-edge-insights-time-series
 >   logs for OPC-UA ingestion after a single data ingestion loop. This message can be ignored.
 > - `make up_opcua_ingestion` is supported only for `Wind Turbine Anomaly Detection` sample app
 
+<!--hide_directive
 ::::{tab-set}
 :::{tab-item} **Wind Turbine Anomaly Detection**
 :sync: tab1
+hide_directive-->
```

Comment:
With this hiding, when one sees the markdown file rendered in github web, they see something like below and we are not differentiating the steps between the sample apps and this may lead to confusion. Is there a way to address this so it works with just some header in markdown and tab in html docs?

<img width="1246" height="645" alt="Image" src="https://github.com/user-attachments/assets/c3c95323-9a3a-4306-8f2c-51620954b4b6" />